go.mod

```diff
@@ -1,18 +1,227 @@
 module github.com/terraform-providers/terraform-provider-venafi
 
-go 1.12
+go 1.21
```

Comment:
This will fail as current pipeline is using default 1.17, make sure to update the pipeline script as well

Reply:
got it